src/fft_simd.rs

```diff
@@ -214,7 +214,18 @@ impl FftSimd<c64> for Scalar {
 
     #[inline(always)]
     fn mul(self, a: c64, b: c64) -> c64 {
-        a * b
+        let ab = a;
+        let xy = b;
```

Comment:
naming looks super confusing, why is this needed ?

Reply:
i just wanted to match the naming from the x86.rs file

Reply:
ok understood